src/index.html

```diff
@@ -20,8 +20,465 @@
+              SMARTPHONES
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="stars stars--0 stars--hidden">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--1 stars--hidden">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--2 stars--hidden">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--3 stars--hidden">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--4 stars--hidden">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--5 stars--hidden">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <main class="container">
+      <div class="catalog">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="Monitor"
+          />
+
+          <h1 class="card__title">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </h1>
```

Comment:
This tag used for title of the full page, so will be better if you use the another tag here
```suggestion
          <h3 class="card__title">
            APPLE A1419 iMac 27" Retina
            <br />
            5K Monoblock (MNED2UA/A)
          </h3>
```